packages/desktopjs/src/Default/default.ts

```diff
@@ -287,21 +283,25 @@ export namespace Default {
 
         protected onOpen(open: (...args: any[]) => Window, ...args: any[]): Window {
             const window = open.apply(this.globalWindow, args);
+            const uuid = Guid.newGuid();
 
             const windows = this.globalWindow[DefaultContainer.windowsPropertyKey];
-            const uuid = window[DefaultContainer.windowUuidPropertyKey] = Guid.newGuid();
             windows[uuid] = window;
```

Comment:
this would now lead dead windows as delete windows[uuid] only runs for same origin

Reply:
Fixed.  Any cross origin windows are no longer tracked.  No layouts, no message bus participation, etc.